lib/pliny/metrics.rb

```diff
@@ -0,0 +1,20 @@
+module Pliny
+  module Metrics
+    def count(*names, value: 1)
+      counts = Hash[names.map { |n| ["count##{Config.app_name}.#{n}", value] }]
+      Pliny.log(counts)
+    end
+
+    def measure(*names, &block)
+      start = Time.now
+      return_value = block.call
+      elapsed = Time.now - start
+
+      measures = Hash[names.map { |n| ["measure##{Config.app_name}.#{n}", elapsed] }]
```

Comment:
If I add it as a private on the module, won't it still contaminate the `Pliny` namespace? That was my main motivator for not extracting a method here.